tests/e2e/conftest.py

```
@@ -18,25 +18,61 @@
+        "docker-compose.yml",
+        "-f",
+        "docker-compose.test.override.yml",
+        "--project-name",
+        test_project_name,
+        "logs",
+    ]
 
     try:
-        # Start the containers
-        compose.start()
+        subprocess.run(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)